### PR TITLE
git: Add git-blame-ignore-revs file

### PR DESCRIPTION
**Pull request by spencer-lunarg**
git 2.23 added a `--ignore-revs-file <file>` option
GitHub picks up this file name by default

### CODE REVIEW

File: .git-blame-ignore-revs [hunk 1]
```diff
@@ -0,0 +1,98 @@
+# Move Device CoreCheck into own file
```
Comment (artem-lunarg) [inline .git-blame-ignore-revs:1]: I can find some entries by the message but not by hash. For example, for this entry git gives this hash 4d196a6c74b7ec9a6f805b7b2959c827a0628cd8

**Review by juan-lunarg — APPROVED** [review]

File: .git-blame-ignore-revs (hunk 1 above)
Comment (ncesario-lunarg) [reply .git-blame-ignore-revs:1]: ^ yeah, this is strange

Comment (spencer-lunarg) [reply .git-blame-ignore-revs:1]: ok, I'm stupid, I went through the PRs and took the commits from there, realized when merged in, these change for what it is in the current main branch... will update date

Comment (spencer-lunarg) [reply .git-blame-ignore-revs:1]: fixed! Thanks for catching this

**Pull request merged**